### ISSUE TITLE
Cache@2 Task documentation is woefully short; there is better information on the general cachinge page for azure pipelines

### ISSUE THREAD

**Issue by richshadman** [MicrosoftDocs/azure-devops-docs#11926]
[Enter feedback here]

This page is missing a lot of relevant information that is available here: https://docs.microsoft.com/en-us/azure/devops/pipelines/release/caching?view=azure-devops#use-cache-task

Specifically the information about how the cache is scoped should be included in the Task documentation as well. One should not have to go searching for other documentation to understand the behavior of the task, which is exactly what we had to do when we were wondering why the cache key did not always result in a "hit" (understanding how the 'scope' works resolved that, but we had to go outside of the Task documentation to figure it out).

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 4422fc0a-f980-9967-1157-d780b644a3ca
* Version Independent ID: adaf52da-a9b7-1057-0c61-d3498b3d3e3c
* Content: [Cache task - Azure Pipelines](https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/utility/cache?view=azure-devops)
* Content Source: [docs/pipelines/tasks/utility/cache.md](https://github.com/MicrosoftDocs/azure-devops-docs/blob/main/docs/pipelines/tasks/utility/cache.md)
* Product: **devops**
* Technology: **devops-cicd-tasks**
* GitHub Login: @damccorm
* Microsoft Alias: **sdanie**

**Comment by damccorm** [comment]
@mmrazik could you help find someone to replace me as the doc owners for these tasks? I think I'm only listed in https://github.com/MicrosoftDocs/azure-devops-docs/blob/main/docs/pipelines/tasks/utility/cache.md and https://github.com/MicrosoftDocs/azure-devops-docs/blob/0dce227aab3898517f3b28fd4ffe59221087232e/docs/pipelines/tasks/includes/yaml/cache-v2.md, but it would be good to have someone with more context tagged for these sorts of PRs

**Comment by mmrazik** [comment]
@damccorm I am not entirely sure what to do. If I look at https://github.com/microsoft/azure-pipelines-tasks/blob/master/.github/CODEOWNERS it seems like it is owned by John? 
Shall I just update the two files you are referring to (you are there as author not necessarily the owner)? Happy to do that.

**Comment by damccorm** [comment]
@mmrazik yeah - that should work I think, I just was not sure who owns them at this point

**Comment by richshadman** [comment]
I know this was auto closed, however I wanted to go on record here that between the time of the original post and now someone else maintaining the documentation did indeed address the missing information on scope - it seems to be now fully defined. 

Thank you!
